mlflow/server/js/src/components/modals/DeleteRunModal.js

```diff
@@ -36,7 +36,22 @@ class DeleteRunModal extends Component {
         onClose={this.props.onClose}
         handleSubmit={this.handleSubmit}
         title={`Delete Experiment ${Utils.pluralize("Run", number)}`}
-        helpText={`${number} experiment ${Utils.pluralize('run', number)} will be deleted.`}
+        helpText={
+          <div>
```

Comment:
Agh, `helpText` is embedded inside a paragraph element, which produces an error because divs cannot be embedded within paragraphs. I will fix this.